app/bingey-ui/src/components/Watchlist/Watchlist.jsx

```diff
@@ -61,16 +65,23 @@ export const Watchlist = ({ titles }) => {
           Add title
         </Button>
       </div>
-      <Grid container spacing={2}>
+      <Grid container spacing={0}>
```

Comment:
If components provide api options for responsive layout, it's better to use them instead of writing them yourself all over again.